src/main/java/com/amazonaws/sagemaker/controller/ServingController.java

```diff
@@ -148,6 +150,54 @@ public ResponseEntity returnBatchExecutionParameter() throws JsonProcessingExcep
         }
     }
 
+    /**
+     * Implements the invocations POST API for application/jsonlines input
+     *
+     * @param jsonLines, lines of json values
+     * @param accept, accept parameter from request
```

Comment:
Accept indicates the content types that the http method is able to understand. Can you change the documentation to indicate that? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

Reply:
Modified